my-tech-journey/genesis.md

```diff
@@ -0,0 +1,39 @@
+## How I got into tech?
+
+To begin with my writing journey, I feel that I must introduce myself, and what's better than to write about my journey in tech and give an insight to what brought me here.
+
+#### The Beginning
+I have been enthusiastic about how the software stuff works, what goes behind the screen of my computer and I always imagined how they're made. I have been keenly interested in computer science since school whether they taught us QBasic, LOGO, VisualC++, SQL, HTML, programming in C++ or networking. Anything related to computer science lured me to read and know more about it. What added to my childhood fantasies was the movie - Jobs(2013) which got me really interested in the life of the tech giant and visionary Steve Jobs. Thereafter, I read several books on his life and wished to be as great as him someday. But as he once said- 
+>"When you grow up, you tend to get told the world is the way that it is, and your life is just to live your life inside the world and try not to bash into the walls too much. But that's a very limited life."
+
+I always wanted to study Computer Science but getting admission in a good college with CS/IT stream is no more an easy deal. You got to almost kill yourself studying Physics, Chemistry and Maths to pass those entrance exams with good marks. I passed the entrance exams I gave but was just on the borderline with the cut-off marks in hand, which led to me a third-tier college affiliated to AKTU, although, I was happy that it was quite famous for it's Computer Science Graduates doing really well with decent placement offers in big firms. Meanwhile, I started writing articles for [fossbytes.com](https://fossbytes.com/author/pragati/).
+
+#### Outset of College
+As I bumped into college life, I got introduced to the cool and awesome technical student societies/clubs of the college, and then it become my prime goal to get in the best of them and try to improve my technical skills and get the best out of what I have got. Moreover, I kept reading books about Elon Musk, Linus Torvalds and Bill Gates. In the first year itself, I was introduced to the world of competitive coding, I thought maybe I can do this. So, I began practicing on platforms like [CodeChef](https://www.codechef.com/users/pragativerma18) and [HackerRank](https://www.hackerrank.com/itispragativerma) etc. It went smooth for sometime, I was able to do 2-3 questions in a long challenge but there came a time I realised that this wasn't I was looking for. I began exploring more options, where I thought that web development might be fun, so I switched to development. 
```

Comment:
Maybe you can expand on some learnings from these books?
How did you realize competitive coding wasn't "it"?

Reply:
Okay, I will add a few more points on that. Thanks for suggestions. 😀